lib/os/__init__.py

```diff
@@ -139,7 +139,7 @@ def close(self, _):
     result = _encode_wait_result(state.Sys())
     if not result:
       return None
-    return state.Pid(), result
+    return state.status 
```

Comment:
Ok, I should update the test, since result would be `None` on successful.
I'm not sure it's a good idea to add a test case for a fail condition (e.g. `command not found` with result `32512`)? Is it going to be equal on different platforms?